src/pages/PeoplePage.tsx

```diff
@@ -0,0 +1,144 @@
+
+  const query = urlParams.get('query');
+  const centuries = urlParams.getAll('centuries');
+  const sortAscParam = urlParams.get('sort') || '';
+  const sortDescParameter = urlParams.get('order');
+
+  const compareQueryMatch = (names: (string | null)[]) => {
+    if (query) {
+      return (
+        names.some(name => {
+          if (name) {
+            return name.toLowerCase().includes(query.toLowerCase());
+          }
+
+          return false;
+        })
+      );
+    }
+
+    return false;
+  };
+
+  const filteredPeople = people.filter(person => {
+    const {
+      name,
+      motherName,
+      fatherName,
+      sex,
+    } = person;
+    const personCentury = (+person.born.toString().slice(0, 2) + 1).toString();
+
+    let isQueryMatch;
+    let isCenturyMatch;
+
+    if (!query || compareQueryMatch([name, motherName, fatherName])) {
+      isQueryMatch = true;
+    }
+
+    if (!centuries.length || centuries.includes(personCentury)) {
+      isCenturyMatch = true;
+    }
+
+    const matchAll = isQueryMatch && isCenturyMatch;
+
+    return urlParams.get('sex')
+      ? sex === urlParams.get('sex') && matchAll
+      : matchAll;
+  });
+
+  const sortedPeople = filteredPeople.sort((personA, personB) => {
+    const key = sortAscParam as keyof Person;
+    const paramHasStringType = typeof personA[key] === 'string';
+
+    switch (paramHasStringType) {
+      case true:
+        return sortDescParameter
+          ? (personB[key] as string).localeCompare(personA[key] as string)
+          : (personA[key] as string).localeCompare(personB[key] as string);
+
+      case false:
+        return sortDescParameter
+          ? (personB[key] as number) - (personA[key] as number)
+          : (personA[key] as number) - (personB[key] as number);
+
+      default: return 1;
+    }
+  });
+
+  useEffect(() => {
+    fetch();
+  }, []);
+
+  return (
+    <>
+      <h1 className="title">People Page</h1>
+
+      <div className="block">
+        <div className="columns is-desktop is-flex-direction-row-reverse">
+          <div className="column is-7-tablet is-narrow-desktop">
+            {people.length > 0 && <PeopleFilters />}
+          </div>
+
+          <div className="column">
+            <div className="box table-container">
+              {isLoading && <Loader />}
+
+              {(!isLoading && !isError && !people.length) && (
+                <p data-cy="noPeopleMessage">
+                  There are no people on the server
+                </p>
+              )}
+
+              {(!isLoading && isError) && (
+                <p
+                  data-cy="peopleLoadingError"
+                  className="has-text-danger"
+                >
+                  Something went wrong
+                </p>
+              )}
+
+              {(!isLoading && sortedPeople.length > 0) && (
+                <PeopleTable people={sortedPeople} />
+              )}
+
+              {(people.length > 0 && !filteredPeople.length) && (
```

Comment:
As official documentation says it is not recommended for React behavior since it will return "0". Please, review 

<img width="901" alt="image" src="https://user-images.githubusercontent.com/104283893/218254405-75cf1f50-0082-4e54-8896-e6a01a82a149.png">

https://reactjs.org/docs/jsx-in-depth.html?#booleans-null-and-undefined-are-ignored